### DISCUSSION TITLE
How to launch rail

### DISCUSSION THREAD

**Discussion by tao363** [FreeRDP/FreeRDP#10434] [Q&A]
I want to use freerdp as proxy to implement remote application, how should I start?

**Comment by akallabeth** [comment]
@tao363 sorry, but what?
could you describe what you want to do a little more verbose?

**Reply by tao363** [reply]
I want to use freerdp as a proxy to give one or more applications from the remote server to the local client instead of giving the entire remote server desktop to the local client

**Reply by akallabeth** [reply]
what do you mean by `proxy`?
normal `RDP RAIL` command line for `xfreerdp`?
your question is still confusing me, not quite sure what you are asking.
